<ui version="4.0" >
 <author>(C) 2003 Andras Mantia &lt;amantia@kde.org></author>
 <comment>***************************************************************************
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; version 2 of the License.               *
 *                                                                         *
 ***************************************************************************
</comment>
 <class>DTEPCreationDlg</class>
 <widget class="QWidget" name="DTEPCreationDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>DTD - > DTEP Conversion</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="nickName" />
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="dtdURL" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Name: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Nickname:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="textLabel3" >
     <property name="text" >
      <string>!DOCTYPE definition line:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="directory" />
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="doctype" />
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="dtdName" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="textLabel5" >
     <property name="text" >
      <string>DTD URL:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="textLabel4" >
     <property name="text" >
      <string>Target directory name:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="textLabel1_2" >
     <property name="text" >
      <string>Default extension:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLineEdit" name="defaultExtension" />
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QCheckBox" name="caseSensitive" >
     <property name="text" >
      <string>Case-sensitive tags and attributes</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QCheckBox" name="fineTune" >
     <property name="text" >
      <string>&amp;Fine-tune the DTEP after conversion</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>directory</tabstop>
  <tabstop>dtdName</tabstop>
  <tabstop>nickName</tabstop>
  <tabstop>doctype</tabstop>
  <tabstop>dtdURL</tabstop>
  <tabstop>defaultExtension</tabstop>
  <tabstop>caseSensitive</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
